tests/unit/ReportSecondaryActionUtilsTest.ts

```diff
@@ -564,7 +564,7 @@ describe('getSecondaryAction', () => {
             violations: {},
             policy,
         });
-        expect(result.includes(CONST.REPORT.SECONDARY_ACTIONS.CANCEL_PAYMENT)).toBe(true);
+        expect(result.includes(CONST.REPORT.SECONDARY_ACTIONS.CANCEL_PAYMENT)).toBe(false);
```

Comment:
We should keep the expectation and update the report statusNum instead, to avoid breaking the test intent
```suggestion
        expect(result.includes(CONST.REPORT.SECONDARY_ACTIONS.CANCEL_PAYMENT)).toBe(true);
```